DEPS

```diff
@@ -371,7 +371,7 @@ deps = {
    Var('chromium_git') + '/webm/libwebp.git' + '@' + '0.6.0',
 
   'src/third_party/wuffs':
-   Var('skia_git') + '/external/github.com/google/wuffs.git' + '@' +  '00cc8a50aa0c86b6bcb37e9ece8fb100047cc17b',
```

Comment:
This was not building with clang 12 so I updated to a version which fixes all the build errors. I am not familiar with this library so let me know if this is not a good version to update to.